cli/command/cli.go

```diff
@@ -118,6 +118,12 @@ func (cli *DockerCli) GetAllCredentials() (map[string]types.AuthConfig, error) {
 	return auths, nil
 }
 
+// GetRegistryCredentials returns the auth config used for a given registry
+func (cli *DockerCli) GetRegistryCredentials(registry string) (types.AuthConfig, error) {
+	helper := cli.CredentialsStore(registry)
+	return helper.Get(registry)
+}
```

Comment:
The `DockerCli` object is passed everywhere so we really need to keep the interface as small as possible. I refactored this a while ago to remove a bunch of these methods.

Instead of adding `DockerCli.GetRegistryCredentials()`, could this be done in the one place it's called?

```go
cli.CredentialsStore(registry).Get(registry)
```